tooling/nargo_cli/tests/compile_success_empty/auto_deref/src/main.nr

```diff
@@ -0,0 +1,8 @@
+
+fn main() {
+    let a = &mut [1, 2, 3];
```

Comment:
Nit: Can we add another array with two or three `& mut`

Reply:
Done